### PR TITLE
Ft secrets injection

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- Updates respective configmaps before making corresponding deployments

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
- You can mainly do static code analysis for these changes
**Or you can:**
- Fork this repository and switch to this branch. 
  `git clone https://github.com/airqo-platform/AirQo-frontend.git`
  `git checkout ft-secretsInjection`
- Add a version to any staging environment secret using the secrets generic accessor account on the GCP Secret Manager.
- Make a change in the respective microservice for which you have just added a secret version
- Commit your changes and create a PR against the staging branch of your fork
- Merge the PR
- After the actions have completed running, checkout the configmap of the microservice you updated
- You should see the changes you made reflecting in that configmap

#### What are the relevant tickets?
- [OPS-39](https://airqoteam.atlassian.net/browse/OPS-39?atlOrigin=eyJpIjoiNzFlMDBjYjI0NTQ1NDdjYmIyYjE2Mjc1YTAxNWZjZDAiLCJwIjoiaiJ9)
- Related to [this PR](https://github.com/airqo-platform/AirQo-api/pull/988) in the api repo

#### Screenshots (optional)

### CODE REVIEW

File: .github/workflows/deploy-frontends-to-production.yml [hunk 1]
```diff
@@ -124,7 +124,22 @@ jobs:
...
+        with:
+          credentials_json: ${{ secrets.GCP_SA_CREDENTIALS }}
+
+      - name: Setup Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0
+
+      - name: Update the corresponding k8s configmap(s)
+        run: |
+          cd netmanager/
+          gcloud secrets versions access latest --secret="prod-env-netmanager" > .env
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:140]: Downloading secret and saving it into a .env file

File: .github/workflows/deploy-frontends-to-production.yml [hunk 2]
```diff
@@ -124,7 +124,22 @@ jobs:
...
+          kubectl create configmap --dry-run=client -o yaml --from-env-file=.env env-platform-production | kubectl replace -f - -n production
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontends-to-production.yml:141]: Updating the configmap on Kubernetes

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**